app/src/main/res/values/styles.xml

```diff
@@ -149,4 +149,18 @@
     <item name="android:windowBackground">@color/cardview_dark_background</item>
   </style>
 
+  <style name="CustomTextView" parent="Widget.Design.TextInputLayout">
```

Comment:
We can do better than "CustomTextView", the style should represent what it is. If I apply this style to a view what does that make this view? If I edit this style what am I changing? CustomTextViews? I don't think so.
`no_list_content_text` I think serves us well here as a name.

If you `ctr+left click` on the parent it will bring you to the declaration
```
    <style name="Widget.Design.TextInputLayout" parent="android:Widget">
    <item name="boxBackgroundMode">none</item>
    <item name="hintTextAppearance">@style/TextAppearance.Design.Hint</item>
    <item name="helperTextTextAppearance">@style/TextAppearance.Design.HelperText</item>
    <item name="errorTextAppearance">@style/TextAppearance.Design.Error</item>
    <item name="counterTextAppearance">@style/TextAppearance.Design.Counter</item>
    <item name="counterOverflowTextAppearance">@style/TextAppearance.Design.Counter.Overflow</item>
    <item name="passwordToggleDrawable">@drawable/design_password_eye</item>
    <item name="passwordToggleTint">@color/design_tint_password_toggle</item>
    <item name="passwordToggleContentDescription">@string/password_toggle_content_description</item>
  </style>
```
Which has attributes that would be good for a TextInputLayout but are not so useful for a TextView. I think `TextAppearance.AppCompat` is the best fit